cheroot/testing.py

```diff
@@ -51,6 +52,7 @@ def cheroot_server(server_factory):
             break
 
     httpserver.shutdown_timeout = 0  # Speed-up tests teardown
+    httpserver.ignored_log_msgs = []  # test-only, log errors to ignore
```

Comment:
please let me know if there's a better/preferred way to make the list of messages to ignore available to the `check_log_messages()` verification step

Reply:
I'd probably say that it's reasonable to have the checks in test functions rather than outside... Except for the generic case. But it's probably okay for starters.